docker_xylem/service.py

```diff
@@ -140,25 +174,38 @@ def get_volume_path(self, request, data):
 
     def remove_volume(self, request, data):
         # FIXME: This probably isn't supposed to do nothing.
-
+        self.log.debug(
+            'Function Call: remove_volume(), `data` = \"{data}\"',
+            data=str(data)
+        )
         return {"Err": None}
 
     @defer.inlineCallbacks
     def create_volume(self, request, data):
+        self.log.debug(
+            'Function Call: create_volume(), `data` = \"{data}\"',
+            data=str(data)
+        )
         name = data['Name']
 
         result = yield self.xylem_request('gluster', 'createvolume', {
             'name': name
         })
 
         if not result['result']['running']:
+            self.log.error('Error creating volume {name}', name=name)
```

Comment:
It would be nice to have more detail here. Do we get anything useful from xylem?